charts/ditto/Chart.yaml

```diff
@@ -31,3 +30,5 @@ maintainers:
   email: axel.koehler@kiwigrid.com
 - name: thjaeckle
   email: thomas.jaeckle@bosch-si.com
+- name: stmaute
+  email: stefan.maute@bosch-si.com
```

Comment:
newline is missing from file - that's what the Helm linter also complains about

Reply:
already fix it